app/javascript/app/components/ndcs/ndcs-commitments/ndcs-commitments.js

```diff
@@ -1,7 +1,22 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import actions from 'pages/ndcs/ndcs-actions';
+import PropTypes from 'prop-types';
+
 import Component from './ndcs-commitments-component';
 import { commitmentsData } from './ndcs-commitments-data';
 
-const NdcsCommitments = () => <Component data={commitmentsData} />;
+class NdcsCommitments extends PureComponent {
+  componentWillMount() {
+    this.props.fetchNDCS(true);
+  }
+
+  render() {
+    return <Component data={commitmentsData} />;
+  }
+}
```

Comment:
Hey, what do you think about starting using hooks? Small components like this are perfect for that ;)